packages/react/src/BlockNoteTheme.ts

```diff
@@ -161,6 +185,48 @@ export const BlockNoteTheme: MantineThemeOverride = {
     },
   },
   fontFamily: "Inter",
+  globalStyles: (theme) => {
+    const textColors: Record<string, any> = {};
+    const backgroundColors: Record<string, any> = {};
+
+    for (const color in theme.other.colorIndex) {
+      textColors[`[data-text-color="${color}"]`] = {
+        color: theme.colors["textColors"][theme.other.colorIndex[color]],
+      };
+      backgroundColors[`[data-background-color="${color}"]`] = {
+        backgroundColor:
+          theme.colors["backgroundColors"][theme.other.colorIndex[color]],
+      };
+    }
+
+    const textAlignments: Record<string, any> = {};
+
+    for (const alignment of ["left", "center", "right", "justify"]) {
+      textAlignments[`[data-text-alignment="${alignment}"]`] = {
+        textAlign: alignment,
+      };
+    }
+
+    return {
+      ...textColors,
+      ...backgroundColors,
+      ...textAlignments,
+    };
+  },
+  other: {
+    colorIndex: {
```

Comment:
Why "colorIndex"? The numbers aren't used right?

Reply:
Oh I was implementing the colors in `BlockNoteTheme` to see how it would work but I think we agreed to do the text/background color styling in the CSS file anyway.

Reply:
but still, why do we need "colorIndex"? can't it be an array for example?